README.md

```diff
@@ -1,45 +1,63 @@
-# newman-orb
+# Orb Project Template
+
+<!---
+[![CircleCI Build Status](https://circleci.com/gh/postmanlabs/newman-orb.svg?style=shield "CircleCI Build Status")](https://circleci.com/gh/postmanlabs/newman-orb) [![CircleCI Orb Version](https://badges.circleci.com/orbs/postman/newman.svg)](https://circleci.com/orbs/registry/orb/postman/newman) [![GitHub License](https://img.shields.io/badge/license-MIT-lightgrey.svg)](https://raw.githubusercontent.com/postmanlabs/newman-orb/master/LICENSE) [![CircleCI Community](https://img.shields.io/badge/community-CircleCI%20Discuss-343434.svg)](https://discuss.circleci.com/c/ecosystem/orbs)
+
+--->
+
 CircleCI Orb (postman/newman) for running Postman collections with Newman - https://github.com/postmanlabs/newman
 
 This Orb is available on CircleCI registry as `postman/newman`
 
 ## Usage
+
 To use the `postman/newman` Orb, reference it in your CircleCI config and then use the `newman-run` command.
-```
+
+```yaml
 version: 2.1
 orbs:
   newman: postman/newman@0.0.2
 jobs:
   newman-collection-run:
     # use the official newman docker image via newman/postman-newman-docker executor
     executor: newman/postman-newman-docker
-    
+
     steps:
       # checkout step is optional if you are not referencing any files from the project
       - checkout
-      
+
       # use newman-run command
       - newman/newman-run:
           collection: ./collection.json
 ```
 
-## Help & Support
-Open a new issue at https://github.com/postmanlabs/newman-orb/issues
+**Meta**: This repository is open for contributions! Feel free to open a pull request with your changes. Due to the nature of this repository, it is not built on CircleCI. The Resources and How to Contribute sections relate to an orb created with this template, rather than the template itself.
 
-## Contributing
-#### Publishing a new version
+## Resources
 
-1. Install [CircleCI CLI](https://github.com/CircleCI-Public/circleci-cli/blob/master/README.md#getting-started)
+[CircleCI Orb Registry Page](https://circleci.com/orbs/registry/orb/postman/newman-orb) - The official registry page of this orb for all versions, executors, commands, and jobs described.
```

Comment:
@kevinswiber I believe the orb registry page should actually go here: https://circleci.com/developer/orbs/orb/postman/newman